packit/cli/scan_in_osh.py

```diff
@@ -0,0 +1,85 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+import os
+from typing import Optional
+
+import click
+
+from packit.cli.types import LocalProjectParameter
+from packit.cli.utils import cover_packit_exception, get_packit_api, iterate_packages
+from packit.config import (
+    PackageConfig,
+    get_context_settings,
+    pass_config,
+)
+from packit.config.aliases import DEPRECATED_TARGET_MAP
+from packit.constants import (
+    PACKAGE_LONG_OPTION,
+    PACKAGE_OPTION_HELP,
+    PACKAGE_SHORT_OPTION,
+)
+from packit.utils import sanitize_branch_name
+from packit.utils.changelog_helper import ChangelogHelper
+
+logger = logging.getLogger(__name__)
+
+
+@click.command("scan-in-osh", context_settings=get_context_settings())
+# @click.option("--wait/--no-wait", default=True, help="Wait for the build to finish")
+# @click.option(
+#     "--description",
+#     help="Description of the project to build in.",
+#     default=None,
+# )
+@pass_config
+@cover_packit_exception
+@iterate_packages
+@click.option(
+    PACKAGE_SHORT_OPTION,
+    PACKAGE_LONG_OPTION,
+    multiple=True,
+    help=PACKAGE_OPTION_HELP.format(action="build"),
+)
+@click.option(
+    "--target",
+    help="Chroot to build in. (defaults to 'fedora-rawhide-x86_64')",
+    default="fedora-rawhide-x86_64",
+)
+@click.option(
+    "--srpm-version-diff",
+    help="Base SRPM to perform a differential build",
```

Comment:
to me calling it `--base-srpm` here would make sense as well, and also mentioning this should be a path in the help message

Reply:
I prefer to keep the names of the commands verbose to avoid any confusion in the future.

@lachmanfrantisek Do you have any other ideas what shall this subcommand be called?

Reply:
Yes, the `base-srpm` sounds reasonable.